cli/src/commands/codegen.rs

```diff
@@ -168,7 +168,8 @@ fn codegen(
             .map_err(|e| eyre!("Cannot parse derive for type {ty_str}: {e}"))?;
         let derive = syn::parse_str(&derive)
             .map_err(|e| eyre!("Cannot parse derive for type {ty_str}: {e}"))?;
-        codegen.add_derives_for_type(ty, std::iter::once(derive));
+        // Note: recursive derives and attributes not supported in the CLI => recursive: false
```

Comment:
Would we want to support recursive derives in the CLI? Might be worth a separate issue to remind us of it

Reply:
Hmm, maybe. Or maybe we just make everything recursive by default in the CLI, I think it is already quite hard to specify derives in the CLI.

Reply:
Would be great if you could open an issue for it :)